### ISSUE TITLE
Turn on CSV URLs for filings and reports

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#2000]
So that users can download valid CSV files of report data, return a `csv_url` field for the `/filings/` and `/reports/` endpoints and enable the field on the front-end.

---

Salient now has valid CSV downloads available. Here is the new CSV link structure, from @PaulClark2 : 
> `http://docquery.fec.gov/csv/XXX/REPID.csv`      (XXX is last 3 digits of the REPID)
> e.g. `http://docquery.fec.gov/csv/488/115488.csv`

> For REPID between 1 and 99 they will be under 000 directory  (They are really old not many people will get there)
 > `http://docquery.fec.gov/csv/000/45.csv`

This is the same scheme for paper and electronic filings.

**Completion criteria**
- [x] Return `csv_url` for all reports that have one on `/filings/`
- [x] Return `csv_url` for all reports that have one on `/reports/**/`
- [x] Enable a "csv" option in the download widget on filings and reports pages

**Comment by noahmanger** [comment]
Update on this: we're no longer blocked on this, as Salient has the URLs ready to go. I updated the issue with further details.

It sounds like this could be a solution for the issue filed here https://github.com/18F/FEC/issues/619

If anyone is finding themselves with downtime and wants to pick this up, I think it's a worthy addition.

**Comment by noahmanger** [comment]
Update #2: turns out all this work was already done on the front-end and the API code just needed to be uncommented, so I have a PR for that.

**Comment by noahmanger** [comment]
Done and shipping.
